andela/travel_tool_back
# #162113738 Make Commands Dependent

#### What does this PR do?
- It makes commands that are associated with development docker container dependent on `make start` command
- It does not stop the application after user executes other make commands

#### Description of Task to be completed?
Currently, when the application is being run using make start command and then make migrate is run in another terminal, it quits the application after migration command is done executing, edit the makefile in such a way that other commands do not quit the application. Also make migrate is independent of make start command, make the make migrate command dependent on make start

#### How should this be manually tested?
- Pull this branch
- Run `make migrate`, it should give you an error.
- Start the application using `make start`, and then run `make migrate` using another terminal, and the migrations should occur.

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
#162113738

#### Screenshots (if appropriate)
<img width="718" alt="screen shot 2018-11-26 at 15 32 51" src="https://user-images.githubusercontent.com/18000724/49014258-9c9e3a00-f190-11e8-8c8a-051a12c331c9.png">


#### Questions:
N/A
